installers/msi-language/Preset/CustomAction.cs

```diff
@@ -230,13 +230,13 @@ public static ActionResult InstallPreset(Session session)
                 var res = preset.Install();
                 if (res != ActionResult.Success)
                 {
-                    RollbarHelper.Report(string.Format("unexpected failure in Preset installation"));
+                    RollbarReport.Error(string.Format("unexpected failure in Preset installation"));
                 }
                 return res;
             }
             catch (Exception err)
             {
-                RollbarHelper.Report(string.Format("unknown error in language preset: {0}", err));
+                RollbarReport.Error(string.Format("unknown error in language preset: {0}", err));
```

Comment:
This should be critical. We want to record the source of the issue as critical, not the tail end, otherwise we still have the same problem.